src/main/com/aca/homework/week11/occurence/KeyCalculator.java

```diff
@@ -0,0 +1,42 @@
+package com.aca.homework.week11.occurence;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class KeyCalculator {
+
+    private final Map<Integer, Integer> occurrencesMap = new HashMap<>();
+    private NumberSupplier numberSupplier;
+
+    public KeyCalculator(NumberSupplier numberSupplier) {
+        this.numberSupplier = numberSupplier;
+    }
+
+    public Integer getValue(int key) {
+        return occurrencesMap.get(key);
+    }
+
+    public void askAndCalculate() {
```

Comment:
the requirement is:

 The method asks a user to input the numbers infinitely until the user writes 'end'
 then it prints and returns a map of a number of occurrences.
 
 
the method should return a map